api/NESSIE-SPEC-2-0.md

```diff
@@ -22,8 +22,15 @@ well-specified behaviours.
 
 Refer to the [Nessie API documentation](./README.md) for the meaning of Nessie-specific terms.
 
+# 2.0.0
+
+* Released with Nessie version 0.59.0.
+* Considered as "GA".
+* Forward/backward breaking changes will no longer happen in 2.0.0.
+
 # 2.0.0-beta.1
 
+* Not compatible w/ spec version 2.0.0.
```

Comment:
Why not?

Reply:
`NessieConfiguration`, you know ;)

Reply:
`2.0.0-beta.1` changed a lot - there are (potentially) breaking changes. Better call it incompatible, so it's clear.